lib/trento_web/openapi/v1/schema/cluster.ex

```diff
@@ -13,6 +13,7 @@ defmodule TrentoWeb.OpenApi.V1.Schema.Cluster do
     OpenApiSpex.schema(
       %{
         title: "ClusterResource",
+        deprecated: true,
```

Comment:
question: are we deprecating both `operations` and `schemas`, aren't we?

Reply:
Yes, I'm deprecating both.
The truth is that deprecating the schemas is not that essential to have the `latest` schema version, as our open api schema generator library excludes schemas that are not used in the included paths.
But I think it is more informative